icons/unity.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img"><title>Unity icon</title><path d="M15.4102 12l4.0273-6.9492L21.3789 12l-1.9414 6.9492zm-1.961 1.129l4.0235 6.9491-7.0079-1.7968-5.0664-5.1524zm4.0235-9.2071l-4.0235 6.9492H5.3984l5.0664-5.1524zm5.746 5.7148L20.7658.5l-9.168 2.4492-1.3555 2.3828-2.754-.0195L.7813 12l6.707 6.6875 2.754-.0195 1.3594 2.3828 9.164 2.4492 2.4532-9.1328L21.8241 12zm0 0"/></svg>
```

Comment:
We'd like the icon to fill its 24x24 viewbox as much as possible. Looks like you have a bit of whitespace around the icon itself here.